airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -318,21 +320,14 @@ public List<CheckedConsumer<JdbcDatabase, Exception>> getCheckOperations(final J
         PostgresUtils.checkFirstRecordWaitTime(config);
       });
 
-      // Verify that the db user has required privilege to perform replication.
+      // Verify that a CDC connection can be created
       checkOperations.add(database -> {
         final String userName = config.get("username").asText();
-
-        final List<JsonNode> userPrivileges = database.queryJsons(connection -> {
-          final PreparedStatement ps = connection.prepareStatement("SELECT userepl FROM pg_user WHERE usename = ?");
-          ps.setString(1, userName);
-          LOGGER.info("Verifying required privileges for user: {}", userName);
-          return ps;
-        }, sourceOperations::rowToJson);
-
-        if (!userPrivileges.get(0).get("userepl").asBoolean()) {
+        try {
+          PostgresDebeziumConnection.createConnection(config);
```

Comment:
We need to close the connection as well so we can put this in try with resources block